dali/pipeline/operator/op_schema.cc

```diff
@@ -199,49 +216,48 @@ OpSchema &OpSchema::AdditionalOutputsFn(SpecFunc f) {
 
 
 OpSchema &OpSchema::NumInput(int n) {
-  DALI_ENFORCE(n >= 0);
+  if (n < 0)
+    throw invalid_key("The number of inputs must not be negative");
```

Comment:
Sounds more like `invalid_argument` than `invalid_key`

Reply:
Nice catch. That should be `out_of_range` - I just replaced too much.